views/detail/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = view.extend({
     id: 'detail',
     template: require('./index.html'),
     data: {
-        back: true,
+        back: '/templates',
```

Comment:
true. I  was originally exploring a hashbang strategy but I can actually put this back in now